lte/gateway/python/integ_tests/s1aptests/test_attach_detach_rar_activation_reject.py

```diff
@@ -21,23 +21,26 @@
 
 
 class TestAttachDetachRarActivationReject(unittest.TestCase):
+    """Unittest: TestAttachDetachRarActivationReject"""
```

Comment:
Let's not call these unit tests, as these require the entire end to end setup for execution
```suggestion
    """Integration test: TestAttachDetachRarActivationReject"""
```